src/pytorch_lightning/accelerators/accelerator.py

```diff
@@ -29,9 +29,10 @@ class Accelerator(_Accelerator, ABC):
     """
 
     def setup_environment(self, root_device: torch.device) -> None:
-        """
-        .. deprecated:: v1.8.0
-            This hook was deprecated in v1.8.0 and will be removed in v1.10.0. Please use ``setup_device()`` instead.
+        """..
+
+        deprecated:: v1.8.0
```

Comment:
Note to self: find out a way to avoid this change

Reply:
Yes, you just need to write any text just before it...